RELEASE_NOTES.md

```diff
@@ -1,13 +1,25 @@
 # Unreleased
 
 Changes:
+
   * Add information for how to fix migration errors in a baremetal install on MariaDB
-  * Allow setting the Active Directory domain DN via `security.ad.domainDn` in `security.properties`. This should only be necessary if you have an unusual ActiveDirectory configuration.
+  * Allow setting the Active Directory domain DN via `security.ad.domainDn` in
+    `security.properties`. This should only be necessary if you have an unusual
+    ActiveDirectory configuration.
+  * New options for searching by date: year, month, fiscal year, fiscal quarter,
+    before/after specified range
+
+Upgrade notes:
+
+  * To enable searching by fiscal year and quarter, you must add the property
+    `miso.fiscalYearStartMonth` to your `miso.properties` file, and set the value to
+    the month number (1-12) of the start of your fiscal year.
```

Comment:
Do you think it's worth noting in the release notes that the `miso.properties` file no longer lives in an `external` directory, on the off chance that there are people who copy that file and `sed` it or something as part of their release process?

Reply:
Not really? The docs have been updated, and it should be easy to find if you had to look for it knowing the previous location